src/style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
If you delete `width` and `justify-content`, then the effect will be almost the same, but this time correct. This is probably causing your problems with tests. Please read checklist again and apply point 3.